src/pages/AddRecord/AddRecordTextArea.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { INPUT_DETAILS } from '../../assets/constant/constant'
+import { CheckAllType, RecordType } from './AddRecord'
+
+type userProps = {
+  currentRecordType: RecordType
+  setCheckAllFilled: (value: any) => void
+  checkAllFilled: CheckAllType
+}
+
+function AddRecordTextArea({
+  currentRecordType,
+  setCheckAllFilled,
+  checkAllFilled,
+}: userProps) {
+  const [textAreaValue, setTextAreaValue] = useState('')
+
+  const handleChangeTextArea = (e: any): void => {
+    if (e.target.value.length > INPUT_DETAILS.MAX_TEXTAREA_TYPING) return
+    if (e.target.value.length > 0)
+      setCheckAllFilled({ ...checkAllFilled, textArea: true })
+    setTextAreaValue(e.target.value)
```

Comment:
이거는 단지 코드스타일인데 저는 한 줄이어도 꼭 `{}`중괄호를 넣는 편입니다! 세인트와 모승은 어떻게 생각하시나요?
맞춰가면 좋을 것 같아요!
```suggestion
    if (e.target.value.length > INPUT_DETAILS.MAX_TEXTAREA_TYPING){
      return
    }
    if (e.target.value.length > 0){
      setCheckAllFilled({ ...checkAllFilled, textArea: true })
    }
    setTextAreaValue(e.target.value)
```

Reply:
ㅋㅋㅋㅋ 프리티어와 중괄호 생략이 만난 요상한 모양이군요 저럴때는 중괄호 넣는게 맞다고 봅니다

Reply:
동의합니다 이것도 코드 스타일 영역이라 저희는 앞으로 모두 중괄호 붙이는걸로 진행해요